source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md

```diff
@@ -0,0 +1,215 @@
+
+- [PHP version](/php-versions#verify-current-php-versions) 7.2 or greater
+
+## Create Your Site
+
+1. Use the [Terminus Build Tools Plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin).
+
+    - The Terminus Build Tools plugin will scaffold the new project, including:
+
+       - A Git repository
+       - A free [Pantheon sandbox](/create-sites#sandbox-sites) site
+       - [Continuous Integration](https://pantheon.io/integrations/continuous-integration) configuration
+       - Credential set up
+
+1. Run the following command to spin up your site: 
+
+    ```bash
+    terminus build:project:create wp
+    ```
+
+    - This site will be based on the Pantheon-maintained [WordPress Composer repository](https://github.com/pantheon-systems/example-wordpress-composer).
+
+1. Review the sections below for important information about your site, including an explanation of the directory structure, Continuous Integration functions, and essential configuration actions.
+
+## Review Important Directories and Update File Paths
+
+### /web Directory
+
+Your site is stored and served from the `/web` subdirectory in the `pantheon.yml` file. You must store your website in this subdirectory for a Composer-based workflow. Placing your website in the subdirectory also allows you  to store tests, scripts, and other files related to your project in your repo without affecting your web document root. It also provides additional security by preventing web access through Pantheon. Your files may still be accessible from your version control project if it is public. See the [`pantheon.yml` documentation](/pantheon-yml#nested-docroot) for details.
+
+1. Verify that your website is stored in the `/web` subdirectory.
+
+### /web/wp Directory
+
+Your directories and files within the `/web` directory are stored in different locations compared to a default WordPress installation. [WordPress allows installing WordPress core in its own directory](https://wordpress.org/support/article/giving-wordpress-its-own-directory/), which is necessary when installing WordPress with Composer. The overall layout of directories in the repo is similar to [Bedrock](https://github.com/roots/bedrock).
+
+1. Move the `WP_SITEURL` to the `/web/wp` directory to allow WordPress core functions to work correctly. 
+
+1. Review the `/web/wp-config.php` file for key settings and move other files to the to the `/web/wp` directory as necessary.
+
+### composer.json File
+
+This project uses Composer to manage third-party PHP dependencies. Some files, such as `web/wp` may not be visible in the repository. This is because WordPress core and its plugins are installed via Composer and ignored in the `.gitignore` file.
+
+A custom, [Composer version of WordPress for Pantheon](https://github.com/pantheon-systems/wordpress-composer/) is used as the source for WordPress core.
```

Comment:
This isn't really a custom version of WordPress; this is a copy of the standard version of WordPress with a composer.json file added. It is maintained automatically via automation.